nyc_taxis/challenges/default.json

```diff
@@ -69,7 +69,7 @@
           "operation": "distance_amount_agg",
           "warmup-iterations": 50,
           "iterations": 100,
```

Comment:
Should we reduce the number of iterations here? If I understand correctly warmup itself takes more than 30mn, that sounds a bit excessive?

Reply:
The variance is so low, I'm inclined to agree this is an excessive number of measurements.